ClientApp/src/modules/common/components/Menu/MenuNav/MenuNav.mocks.ts

```diff
@@ -4,4 +4,4 @@ export enum UserType {
     INSURANCE_EMPLOYEE,
 }
 
-export const CURRENT_USER = UserType.WORKSHOP_EMPLOYEE;
```

Comment:
co z employee?

Reply:
To jest tylko mock który wybiera które menu mamy wyświetlać. Zmieniłem na Client, żeby odpowiadało temu co mockujesz w autentykacji